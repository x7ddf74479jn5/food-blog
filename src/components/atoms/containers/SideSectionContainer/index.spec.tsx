import renderer from "react-test-renderer";

import { SideSectionContainer } from ".";

describe("components/atoms/containers/SideSectionContainer", () => {
  it("snapshot", () => {
    const tree = renderer.create(<SideSectionContainer header="header">children</SideSectionContainer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
