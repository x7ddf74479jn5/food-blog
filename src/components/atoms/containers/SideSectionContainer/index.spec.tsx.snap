// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/containers/SideSectionContainer snapshot 1`] = `
<section
  className="container p-2 dark:text-white bg-gray-50 dark:bg-gray-700"
>
  <h2
    className="px-2 pb-2 mb-2 border-b-2"
  >
    header
  </h2>
  children
</section>
`;
